src/Program.cs

```diff
@@ -21,6 +22,17 @@ public static class Program
     {
         private static readonly HttpClient _httpClient = new HttpClient();
 
+        private static IList<KeyValuePair<string, string>> _selectionDescriptions = new List<KeyValuePair<string, string>>()
```

Comment:
```suggestion
        private static readonly List<KeyValuePair<string, string>> _selectionDescriptions = new List<KeyValuePair<string, string>>()
```

NIT: Make readonly and use class instead of interface.